prototype_source/fx_graph_mode_ptq_static.rst

```diff
@@ -54,14 +54,9 @@ Advantages of FX Graph Mode Quantization are:
   
 We’ll start by doing the necessary imports, defining some helper functions and prepare the data.  
 These steps are identitcal to `Static Quantization with Eager Mode in PyTorch <https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html>`_.       
-  
-Download dataset: 
-  
-.. code:: 
-  
-    wget https://s3.amazonaws.com/pytorch-tutorial-assets/imagenet_1k.zip 
-  
-and unzip to `data` folder.   
+
+To run the code in this tutorial using the entire ImageNet dataset, first download imagenet by following the instructions at here `ImageNet Data <http://www.image-net.org/download>`_. Unzip the downloaded file into the 'data_path' folder.
+
```

Comment:
Are we asking the reader to use a pretraining model or download and train on full imagenet? It looks like both at this point.

Reply:
reader will use a pretrained model and do post training quantization, we are downloading the full imagenet just for evaluating the accuracy

Reply:
got it - thanks Jerry. LGTM..